components/locid/src/extensions/mod.rs

```diff
@@ -150,13 +150,13 @@ impl Extensions {
     /// only_unicode
     ///     .extensions
     ///     .retain_by_type(|t| t == ExtensionType::Unicode);
-    /// assert_eq!(only_unicode, "und-u-world");
+    /// assert_eq!(only_unicode, "und-u-world".parse().unwrap());
```

Comment:
Suggestion (optional, here and elsewhere): I would slightly prefer

```suggestion
    /// assert_eq!(only_unicode.to_string(), "und-u-world");
```

I see you did this in some places but not others.

Reply:
I think I only did this in places where there was already a check for structural equality.